data/story oathkeepers.txt

```diff
@@ -0,0 +1,168 @@
+	cargo "sensor equipment" 5
+	passengers 3
+	on offer
+		conversation
+			`As you're passing a local restaurant, a familiar face approaches. Captain Jane Mulgrew calls over to you, "Hey, Captain <last>! It's good to see you again. I heard you became a contractor, and that's even better! We're short-handed with the war and all, so we can always use good pilots."
+			`	"Let's get right into it, shall we? The hyperspace relay system here is in a dreadful state, and I'd like some help fixing it up, so that the Oathkeepers can keep up with the Northern Union anarchists around here. For now, I just need you to take me and my assistants to the Hassaleh system with our equipment so we can install a sensor. It is a little bit out of the way, but I'm sure you're used to that."`
+				choice
+					`	"Sure, I'll go fire up the engines."`
+					`	"Can it wait? I'm a little busy right now."`
+						defer
+				`	"Great! I'll get everything loaded up and we can get to work."`
+					accept
+	on accept
+		log "Met Jane Mulgrew at Farpoint, agreed to take her to Hassaleh to install sensors."
+	on visit
+		dialog `Mulgrew and her crew install the sensor on a large asteroid from the system's belt, and it seems to be working. Time to go back to Farpoint.`
+	on complete
+		dialog `Mulgrew says the sensor is working like a dream and thanks you, then promises to be back in the spaceport with more work later.`
+		
+mission "Oathkeeper Scouting 1"
+	name `Put surveillance equipment on Ada`
+	description `The Navy suspects the Northern Union will take advantage of the war with the Free Worlds to raid Ada. Mulgrew sent you to install bugs, just in case.`
+	to offer
+		has "Oathkeeper Intro: complete"
+	source "Farpoint"
+	destination "Ada"
+	on offer
+		conversation
+			`Mulgrew called you to meet her at a little cafe in the spaceport, and she wastes no time: "Good news, Captain! I went into a meeting recently and we have more work! The Northern Union is taking advantage of the war down south, but they're still pretty outgunned, and they know it."
+			choice
+				`	"Do you think they're going to try to steal our weapons?"`
+				`	"So, are we attacking?"`
+					goto attack
+			`	"Yes, precisely, very observant!" Mulgrew grins and continues, "We believe they're going to try and overwhelm us with multiple hit-and-run attacks on our shipyards and outfitters, so we need to set up some emergency defenses. I'd like you to install some surveillance equipment on a couple of Lovelace Labs planets. Start with Ada and we can work from there."
+			label attack
+			`	"We being the Navy? No, at least not yet. War does cost a lot, even up here. We've been containing the Northern Union for decades, and are currently ready to make a push. At least, not with so many gaps in our defense. Speaking of which, I'd like your help with that! Could you put some listening devices on Ada just in case the Northern Union raids them for weapons?"

```

Comment:
```suggestion
			`	"We being the Navy? No, at least not yet. War does cost a lot, even up here. We've been containing the Northern Union for decades, and are currently ready to make a push. At least, not with so many gaps in our defense. Speaking of which, I'd like your help with that! Could you put some listening devices on Ada just in case the Northern Union raids them for weapons?"`

```